Sources/JWT/JWTDecoder.swift

```diff
@@ -11,7 +11,7 @@ internal extension JWT.Token {
     var isExpired: Bool {
         do {
             let decodedBearer = try JWTDecoder.decode(self)
-            return decodedBearer.expiryDate.compare(Date()) != ComparisonResult.orderedDescending
+            return decodedBearer.expiryDate.timeIntervalSince(Date()) < 60 * 5
```

Comment:
Can you explain why you did this change? Why is the expired date not correct without the extra interval `60 * 5`?